packages/simcore-sdk/src/simcore_sdk/node_ports_common/r_clone.py

```diff
@@ -30,7 +56,25 @@ async def _config_file(config: str) -> AsyncIterator[str]:
         yield f.name
 
 
-async def _async_command(*cmd: str, cwd: str | None = None) -> str:
+async def _read_stream(
+    stream: StreamReader, r_clone_log_parsers: list[BaseRCloneLogParser]
+):
+    while True:
+        line: bytes = await stream.readline()
+        if line:
+            decoded_line = line.decode()
+            await logged_gather(
+                *[parser(decoded_line) for parser in r_clone_log_parsers]
+            )
+        else:
+            break
+
+
+async def _async_command(
```

Comment:
this should probably go in async_utils in servicelib or so...

Reply:
this is very specific to how we call rclone here, will rename it.